### ISSUE TITLE
"Ambiguous class resolution" on phpunit bridge since symfony 3.0.4

### ISSUE THREAD

**Issue by luispabon** [symfony/symfony#18435]
Since upgrading my symfony installation to 3.0.4 from 3.0.3, I get the following when running composer install:

```
[development@localhost ~/P/phpdocker.io] (behat) composer -o install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. Run update to update them.
Nothing to install or update
Generating optimized autoload files
Warning: Ambiguous class resolution, "Symfony\Bridge\PhpUnit\ClockMock" was found in both "/home/development/Projects/phpdocker.io/vendor/symfony/phpunit-bridge/ClockMock.php" and "/home/development/Projects/phpdocker.io/vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/ClockMock.php", the first will be used.
Warning: Ambiguous class resolution, "Symfony\Bridge\PhpUnit\TextUI\Command" was found in both "/home/development/Projects/phpdocker.io/vendor/symfony/phpunit-bridge/TextUI/Command.php" and "/home/development/Projects/phpdocker.io/vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/TextUI/Command.php", the first will be used.
Warning: Ambiguous class resolution, "Symfony\Bridge\PhpUnit\TextUI\TestRunner" was found in both "/home/development/Projects/phpdocker.io/vendor/symfony/phpunit-bridge/TextUI/TestRunner.php" and "/home/development/Projects/phpdocker.io/vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/TextUI/TestRunner.php", the first will be used.
Warning: Ambiguous class resolution, "Symfony\Bridge\PhpUnit\SymfonyTestsListener" was found in both "/home/development/Projects/phpdocker.io/vendor/symfony/phpunit-bridge/SymfonyTestsListener.php" and "/home/development/Projects/phpdocker.io/vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php", the first will be used.
Warning: Ambiguous class resolution, "Symfony\Bridge\PhpUnit\DeprecationErrorHandler" was found in both "/home/development/Projects/phpdocker.io/vendor/symfony/phpunit-bridge/DeprecationErrorHandler.php" and "/home/development/Projects/phpdocker.io/vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php", the first will be used.
ocramius/package-versions:  Generating version class...
ocramius/package-versions: ...done generating version class
ocramius/package-versions:  Generating version class...
ocramius/package-versions: ...done generating version class
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::buildBootstrap
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installRequirementsFile
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::prepareDeploymentTarget
```

This doesn't happen if I force symfony version to 3.0.3.

composer.json:

```json
{
  "name":         "development/php-docker",
  "license":      "proprietary",
  "type":         "project",
  "autoload":     {
    "psr-4":    {
      "": "src/"
    },
    "classmap": [
      "app/AppKernel.php",
      "app/AppCache.php"
    ]
  },
  "autoload-dev": {
    "psr-4": {
      "Tests\\": "tests/"
    }
  },
  "require":      {
    "php":                             ">=7.0.0",
    "symfony/symfony":                 "3.0.*",
    "symfony/monolog-bundle":          "^2.8",
    "symfony/assetic-bundle":          "^2.7",
    "symfony/swiftmailer-bundle":      "^2.3",
    "doctrine/orm":                    "^2.5",
    "doctrine/doctrine-bundle":        "^1.6",
    "doctrine/doctrine-cache-bundle":  "^1.2",
    "stof/doctrine-extensions-bundle": "^1.2",
    "sensio/distribution-bundle":      "^5.0",
    "sensio/framework-extra-bundle":   "^3.0.2",
    "cocur/slugify":                   "^1.4",
    "michelf/php-markdown":            "^1.6",
    "guzzlehttp/guzzle":               "^6.1",
    "javiereguiluz/easyadmin-bundle":  "^1.11",
    "egeloen/ckeditor-bundle":         "^4.0",
    "doctrine/doctrine-migrations-bundle": "^1.1"
  },
  "require-dev":  {
    "sensio/generator-bundle": "^3.0",
    "symfony/phpunit-bridge":  "^2.7",
    "symfony/var-dumper":      "^3.0",
    "behat/behat": "^3.1",
    "behat/symfony2-extension": "^2.1",
    "behat/mink": "^1.7",
    "behat/mink-extension": "^2.2",
    "behat/mink-browserkit-driver": "^1.3"
  },
  "scripts":      {
    "post-install-cmd": [
      "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
      "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
      "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
    ],
    "post-update-cmd":  [
      "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
      "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
      "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
    ]
  },
  "extra":        {
    "symfony-app-dir":        "app",
    "symfony-bin-dir":        "bin",
    "symfony-var-dir":        "var",
    "symfony-web-dir":        "web",
    "symfony-tests-dir":      "tests",
    "symfony-assets-install": "relative"
  }
}
```

**Comment by nicolas-grekas** [comment]
Thanks for the report.
Closing as duplicate of #18381, fixed in #18389.

**Comment by luispabon** [comment]
:+1:
